docs/docs/references/logto-sdk-convention/README.mdx

```diff
@@ -0,0 +1,9 @@
+# 📃 Logto SDK Convention
+
+This section demonstrates functionalities provided by Logto SDK and the convention between various languages and platforms.
```

Comment:
this article?

Reply:
Sorry for the outdated snapshot.
In the latest version, this has become a 'section'.